ksqldb-parser/src/main/java/io/confluent/ksql/parser/tree/AstVisitor.java

```diff
@@ -180,4 +181,16 @@ public R visitRegisterType(final RegisterType node, final C context) {
   public R visitDropType(final DropType node, final C context) {
     return visitStatement(node, context);
   }
+
+  public R visitAssertValues(final AssertValues node, final C context) {
+    return visitStatement(node.getStatement(), context);
+  }
+
+  public R visitAssertStream(final AssertStream node, final C context) {
+    return visitStatement(node.getStatement(), context);
+  }
+
+  public R visitAssertTable(final AssertTable node, final C context) {
+    return visitStatement(node.getStatement(), context);
+  }
```

Comment:
What happens ASSERT is used in production ksqlDB?  Worth adding a QTT test case?

Reply:
yeah, production code doesn't use the `testStatement` parser rule so it shouldn't find it. I'll add a test in the follow-up PR (or if the build isn't green, don't want to wait another 45min jenkins cycle 😄 )